mint-auditor/src/db/schema.rs

```diff
@@ -0,0 +1,26 @@
+table! {
+    block_audit_data (id) {
+        id -> Nullable<Integer>,
```

Comment:
Ah, I didn't know that. Will we need to regenerate this file for schema changes? Either way, documenting that in this file seems helpful.

Reply:
Yes, when migrations are edited part of the process is regenerating the file. I don't want to make edits to it since they will get removed every time it is regenerated, and there's no reason a human should look at this file.

Reply:
I'll put the comment in the migrations file instead